docs/eventing/sink/kafka-sink.md

```diff
@@ -15,13 +17,13 @@ This page shows how to install and configure Apache Kafka Sink.
 1. Install the Kafka controller:
 
     ```bash
-    kubectl apply --filename {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-controller.yaml" >}}
+    kubectl apply --filename {{ artifact(org="knative-sandbox", repo="eventing-kafka-broker", file="eventing-kafka-controller.yaml") }}
```

Comment:
Why does this work?!

Reply:
https://github.com/knative/docs/blob/4738fe3b8e2605c9d0dff6b89a3d3b2601b4db8f/hack/macros.py#L6